go/obscuronode/enclave/sql/sql.go

```diff
@@ -0,0 +1,224 @@
+package sql
+
+import (
+	"database/sql"
+	"errors"
+	"fmt"
+
+	"github.com/obscuronet/obscuro-playground/go/log"
+
+	"github.com/ethereum/go-ethereum/ethdb"
+)
+
+// sqlEthDatabase implements ethdb.Database
+type sqlEthDatabase struct {
+	db *sql.DB
+}
+
+func CreateSQLEthDatabase(db *sql.DB) (ethdb.Database, error) {
+	s := &sqlEthDatabase{db: db}
+	if err := s.Initialise(); err != nil {
+		return nil, err
+	}
+	return s, nil
+}
+
+func (m *sqlEthDatabase) Initialise() error {
+	stmt := `create table if not exists kv (key text primary key, value blob); delete from kv;`
```

Comment:
does this mean that every time the node is started it will wipe the db?
This is ok for tests, but we need a different mechanism

Reply:
Yeah sorry, I'll move this now to the sqlite part of the code. You can create tables in the 'manifest' that you give to the edgeless DB node when it's started so for prod it will be in there.